<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PstoeditQtGuiClass</class>
 <widget class="QMainWindow" name="PstoeditQtGuiClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1026</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PstoeditQtGui</string>
  </property>
  <property name="windowIcon">
   <iconset resource="PstoeditQtGui.qrc">
    <normaloff>:/PstoeditQtGui/pstoeditlogo-medium.ico</normaloff>:/PstoeditQtGui/pstoeditlogo-medium.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="minimumSize">
    <size>
     <width>745</width>
     <height>435</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab_ifo">
       <attribute name="title">
        <string>Input, Format, and Output Settings</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QWidget" name="formWidget" native="true">
          <layout class="QFormLayout" name="formLayout"/>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_log">
       <attribute name="title">
        <string>Conversion Log</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QScrollArea" name="scrollArea">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>982</width>
             <height>368</height>
            </rect>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QPlainTextEdit" name="logWidget"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1026</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionOpen_Help_Manual"/>
    <addaction name="actionAbout"/>
    <addaction name="actionSupport_Pstoedit_Development_and_Maintenance"/>
    <addaction name="actionCheck_for_Updates"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSelectFiles"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen_Help_Manual">
   <property name="text">
    <string>Pstoedit Documentation</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About Pstoedit</string>
   </property>
  </action>
  <action name="actionSelectFiles">
   <property name="text">
    <string>Select file(s) to be converted</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionSupport_Pstoedit_Development_and_Maintenance">
   <property name="text">
    <string>Support Pstoedit Development and Maintenance</string>
   </property>
  </action>
  <action name="actionCheck_for_Updates">
   <property name="text">
    <string>Check for Updates</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="PstoeditQtGui.qrc"/>
 </resources>
 <connections/>
</ui>
